lib/linter/timing.js

```diff
@@ -32,7 +32,7 @@ function alignLeft(str, len, ch) {
  * @private
  */
 function alignRight(str, len, ch) {
-    return new Array(len - str.length + 1).join(ch || " ") + str;
+    return (ch || " ").repeat(len - str.length) + str;
```

Comment:
Isn't this `String#padStart()`?

Reply:
I think this and others may work well to be more succinct, but I think to avoid scope creep, we should stick in this PR with the auto-fixes provided by the linter. Looking for these patterns in another PR might also more systematically identify patterns that could be improved elsewhere in the codebase.